jdaviz/configs/default/plugins/export/export.py

```diff
@@ -200,11 +218,27 @@ def export(self, filename=None, show_dialog=None):
                 self.save_movie(viewer, filename, filetype)
             else:
                 self.save_figure(viewer, filename, filetype, show_dialog=show_dialog)
+
         elif len(self.table.selected):
             filetype = self.table_format.selected
             if not filename.endswith(filetype):
                 filename += f".{filetype}"
             self.table.selected_obj.export_table(filename, overwrite=True)
+
+        elif len(self.table.selected):
+
+            if "." not in filename:
+                filename += ".ecsv"
+            self.table.selected_obj.export_table(filename, overwrite=True)
+

```

Comment:
Looks like the rebase duplicated this.  The previous elif is the latest version.

```suggestion
```

Reply:
done